src/components/DataCollector/index.js

```diff
@@ -85,6 +87,9 @@ createDataCollector.configValidators = {
   },
   imsOrgId: {
     validate: required
+  },
+  clickCollectionEnabled: {
```

Comment:
this is `linkClickCollection` right, or will it expand into clickCollection?

Reply:
It is link-click-collection. I didn't want to reduce the scope and kept it as click. But I also don't want to make it misleading. Once we know the other properties that it needs to collect we can revisit the naming convention.